tests/unittests/config/test_cc_landscape.py

```diff
@@ -210,6 +224,37 @@ def test_handler_writes_merged_provided_cloudconfig_with_defaults(
         )
         assert expected_calls == m_subp.call_args_list
 
+    @mock.patch(f"{MPATH}.merge_together")
+    def test_handler_client_failed_registering(self, m_subp, m_merge_together):
```

Comment:
The decorator ordering is flipped, which results in the unittest errors here.
```suggestion
    def test_handler_client_failed_registering(self, m_merge_together, m_subp):
```

Reply:
thanks for the fix and explanation :)